### DISCUSSION TITLE
Changing a `ListViewItem` height

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#11010] [Q&A] [answered]
[_From Discord_:](https://discord.com/channels/372137812037730304/550416151172087808/1065633938606915636)

Anyone have an idea how to fix the height of a ListViewItem for WASM ?   (Left = UWP, Right = Wasm)

![image](https://user-images.githubusercontent.com/5839577/213467980-92d10e29-47e8-4cbb-bddb-ba7250494037.png)

![image](https://user-images.githubusercontent.com/5839577/213468020-cd7dd584-5710-4894-beee-db55c80f0cfe.png)

**Comment by jeromelaban** [comment, marked as answer]
If you're using the default Fluent style, there's a padding in the default template that may want to change:

https://github.com/unoplatform/uno/blob/38b87e6d2e0da5c945de2a6b3858c392ce7fd2a3/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/ListViewItem_themeresources.xaml#L241

Are you using Fluent v2 styles in your UWP app ?

**Reply by assassin316** [reply]
I believe I'm on V2, I'll double check. However, the padding listed above is on the sides and not the top/bottom. I'll dig into this style a little more and see what I can do. Thanks!

**Reply by jeromelaban** [reply]
Indeed, my bad. There's likely something else in the styles that is incorrect (or different). Are you able to reproduce this in the playground?

**Reply by assassin316** [reply]
Fixed!   I was able to reproduce it in the Playground. By specifying both `MinHeight` and `Height`, I was able to fix the issue. I used the following code in the Playground:

```xml
<ListView ItemsSource="{Binding Fonts}"
              x:Name="fontPicker">
        <ListView.ItemContainerStyle>
            <Style TargetType="ListViewItem">
                <Setter Property="HorizontalContentAlignment"
                        Value="Stretch" />
                <Setter Property="VerticalContentAlignment"
                        Value="Center" />
                <Setter Property="MinHeight"
                        Value="26" />
                <Setter Property="Height"
                        Value="26" />
            </Style>
        </ListView.ItemContainerStyle>
    </ListView>
```

Thanks for the help!
